tests/devices/unit_tests/test_bart_robot.py

```diff
@@ -94,24 +100,34 @@ async def test_given_program_not_running_and_pin_unmounting_but_new_pin_not_moun
         raise
 
 
-async def set_with_happy_path(device: BartRobot) -> AsyncStatus:
+def _set_pin_sensor_on_log_messages(device: BartRobot, msg: str):
```

Comment:
Should: If we have to do this then lets at least pull them into constants so that the relationship is more obvious